sympy/calculus/util.py

```diff
@@ -253,6 +254,192 @@ def elm_domain(expr, intrvl):
         return _domain
 
 
+def periodicity(f, symbol, check=False):
```

Comment:
> Can you simply call the function period?

I had named it `periodicity` because:

1. `period` is an instance method of class `TrigonometricFunction`. Having two functions with the same name(and same behaviour) might lead to confusion in the future.

2. I intend to read this function as "Check the function `f` for periodicity in `symbol`. If it is periodic, return the value of the period."